src/main/resources/templates/email/TechnicalQuestions_Interviewer.txt

```diff
@@ -0,0 +1,14 @@
+Hello {{interviewer_first_name}},
+
+We see that you will be giving a {{position}} mock interview with InterviewMe on {{formatted_date}}. These interviews usually come with technical questions, which you should use to measure the candidate's competency. We wanted to share with you some tips on handling this technical portion.
+
+  - Leave some vagueness in the question. This gives the candidate the opportunity to ask clarifying questions.
+  - Does the candidate have the right approach in mind? Or is theirs going to lead them in the wrong direction? If you feel they are on the wrong track, hint them so.
```

Comment:
Avoid theirs, maybe replace with "their approach". Or else "Or will it lead..." or "Or is it going..."

Reply:
"Hint them so" is weird. Replace with "let them know" or "provide a hint that will bring them in the right direction".